lib/internal/webstreams/readablestream.js

```diff
@@ -1471,7 +1470,7 @@ function readableStreamDefaultTee(stream, cloneForBranch2) {
           let value2 = value;
           if (!canceled2 && cloneForBranch2) {
             // Structured Clone
```

Comment:
The code comment is now moot as it just says the same as the function name.

Reply:
Removed and pushed `fixup` commit.